modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceFlavor.java

```diff
@@ -34,4 +34,7 @@ public class ServiceFlavor implements Serializable {
     @Schema(description = "The properties of the flavor")
     private Map<String, String> properties;
 
+    @NotNull
+    @Schema(description = "The priority of the flavor. The larger value means higher priority.")
+    private int priority;
```

Comment:
would be good if we can have a check that priority minimum allowed value is "1"

Reply:
Updated.